packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -378,6 +387,15 @@ const GroupForm = ({
               placeholder="Group name"
               fullWidth
               instructionalMessage="Can be up to 40 characters long."
+              customError={isNameTaken ? 'Group name is already taken' : ''}
+              onInput={(e) => {
+                const value = e.target.value.toLowerCase();
+                if (takenGroupNames.includes(value)) {
+                  setIsNameTaken(true);
+                } else {
+                  setIsNameTaken(false);
+                }
+              }}
```

Comment:
i mostly try to avoid state variables in forms as they snowball pretty quickly. Maybe we can try this
```jsx
customError={takenGroupNames.includes(formState?.getValues?.()?.groupName?.trim()) ? 'Group name is already taken' : ''}
```

also if this sounds bad, plz feel free to keep current change.

Reply:
This won't work because it needs to be reactive to the `onChange` or in this scenario the `onInput`, i like the idea of watching out for future snowballing but here i don't think theres a real risk.

Reply:
i see, then i also agree to keep current change